### DISCUSSION TITLE
java.net.SocketException: Broken pipe (Write failed)

### DISCUSSION THREAD

**Discussion by calvin2020** [apache/doris#33601] [A - General / Q&A]
Hi:
My doris fe version is 1.2.3,I get the error as attachment,I have no idea,would you please help me to check it
thanks
[doris_fe_logs.txt](https://github.com/apache/doris/files/14957964/doris_fe_logs.txt)

**Comment by yagagagaga** [comment]
The information provided is too scarce, maybe you can join community and have a deeper communication. https://github.com/apache/doris/discussions/16887
